docs/UserGuide.md

```diff
@@ -65,17 +67,41 @@ command to run the application.<br>
 
 Adds a client to Realodex. 
 
```

Comment:
Perhaps you can consider adding a note saying that if user makes a mistake, the error message will let you know exactly what is wrong with your input.

Reply:
Yup that is in line 77

Reply:
psps never see